commit b5f729c1b14ad771cf75fc4ea200df0567fc05aa
Author: Matthieu Herrb <matthieu.herrb@laas.fr>
Date:   Mon Mar 1 17:15:51 2010 +0100

    Replace Togl_Visual() with Tk_Visual().
    
    The former doesn't exist in recent Togl versions anymore.

diff --git modules/planet/planet.c modules/planet/planet.c
index 9c2f8bc..f05900d 100644
--- modules/planet/planet.c
+++ modules/planet/planet.c
@@ -181,7 +181,7 @@ static void
 setup_file_texture (struct Togl *togl, char *filename)
 {
     Display *dpy = Togl_Display(togl);
-    Visual *visual = Togl_Visual(togl);
+    Visual *visual = Tk_Visual(Togl_TkWin(togl));
     Colormap cmap = Togl_Colormap(togl);
 
     char **xpm_data = 0;
